p2panda-net-next/src/actors/subscription.rs

```diff
@@ -25,6 +44,8 @@ pub struct SubscriptionState {
     gossip_actor_failures: u16,
     sync_actor: ActorRef<ToSync>,
     sync_actor_failures: u16,
+    ephemeral_senders: HashMap<TopicId, Sender<ToNetwork>>,
+    gossip_senders: HashMap<TopicId, BroadcastSender<FromNetwork>>,
```

Comment:
Slightly hard to grok the relation between these two maps from the naming, some kind of `to_network_senders` and `from_network_senders` could be better (not sure about the "network" part...).